idaes/core/util/model_diagnostics.py

```diff
@@ -291,6 +293,14 @@ def svd_sparse(jacobian, number_singular_values):
         description="Tolerance for identifying near-parallel Jacobian rows/columns",
     ),
 )
+CONFIG.declare(
+    "absolute_feasibility_tolerance",
+    ConfigValue(
+        default=1e-6,
+        domain=float,
+        description="Feasiblity tolerance for idenifying infeasible constraint and bounds",
```

Comment:
Typo: "idenifying"

Reply:
Also "feasiblty" I believe

Reply:
It should probably also be constraint**s** (plural).

Reply:
fixed in [34668b0](https://github.com/IDAES/idaes-pse/pull/1409/commits/34668b0d30b1dac4c0fe905abf11e73b355a48a4) and [40670e4](https://github.com/IDAES/idaes-pse/pull/1409/commits/40670e4dc0d66a766d87d4f70fb7820ecbf30965)